acceptance/testdata/builder/container/env_folders/different_buildpack_dir_from_env/hello_world/0.0.1/bin/build

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+echo "---> Hello World buildpack"
```

Comment:
Same comment as https://github.com/buildpacks/lifecycle/pull/648/files#r669969357 (that will also make the duplication less painful)

Reply:
deleted most parts